### PR TITLE
OS3 test fixes in 11.0.0.GA

### PR DESCRIPTION
**Pull request by rhopp**
Signed-off-by: rhopp <rhopp@redhat.com>

### CODE REVIEW

**Comment by rhopp** [comment]
@jkopriva  review pls ;-)

**Comment by jkopriva** [comment]
looks good to me, +1, @psrna @rawagner please merge

File: tests/org.jboss.tools.openshift.ui.bot.test/resources/eap64-basic-s2i.json [hunk 1]
```diff
@@ -30,13 +30,13 @@
         {
             "description": "Git source URI for application",
             "name": "SOURCE_REPOSITORY_URL",
-            "value": "https://github.com/mlabuda/jboss-eap-quickstarts",
+            "value": "https://github.com/rhopp/jboss-eap-quickstarts",
```
Comment (psrna) [inline tests/org.jboss.tools.openshift.ui.bot.test/resources/eap64-basic-s2i.json]: Why not use: https://github.com/jboss-developer/jboss-eap-quickstarts?

Or is there something modified in the quickstarts we use?

Comment (jkopriva) [inline tests/org.jboss.tools.openshift.ui.bot.test/resources/eap64-basic-s2i.json]: One quickstart have modified pom.xml to include openshift profile - I am planning to get rid of this dependency...

Comment (jkopriva) [inline tests/org.jboss.tools.openshift.ui.bot.test/resources/eap64-basic-s2i.json]: Jira for tracking this: https://issues.jboss.org/browse/JBIDE-24867

**Pull request merged**